Authorization/Authorization.Domain/Enums/Roles.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Authorization.Domain.Enums
+{
+    public enum Roles
```

Comment:
Use singular enum name instead of Plural
Check this out: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Reply:
Fixed